kafka-connect/kafka-connect-events/src/main/java/org/apache/iceberg/connect/events/DataWritten.java

```diff
@@ -49,7 +49,7 @@ public class DataWritten implements Payload {
   static final int DATA_FILES = 10_302;
   static final int DATA_FILES_ELEMENT = 10_303;
   static final int DELETE_FILES = 10_304;
-  static final int DELETE_FILES_ELEMENT = 10_304;
+  static final int DELETE_FILES_ELEMENT = 10_305;
```

Comment:
@bryanck this seems like a mistake?

Not quite sure what the implications are TBH.

Reply:
Thanks for catching this! I don't believe there is an impact currently w/ the serialization we're using.